cap-ci/pipeline.template

```diff
@@ -275,6 +275,64 @@ jobs:
             {{- end }}
             {{- print $test }}
 
+- name: cf-acceptance-tests-brain-{{$EiriniFlag}}-{{$Backend}}-{{$OptionsFlag}}
```

Comment:
Nitpicking: could the `cf-acceptance-tests-brain` job block go below the `cf-acceptance-tests` one? that way the tasks are ordered as in the pipeline.

Reply:
Sure, that is no problem at all. The physical ordering is a leftover from the initial logical order. Then I moved the BT towards the end, after the SITS.

Reply:
Done with commit 8f9c24436aa56d082e5c2266fbb33e75c8af7731.